test/unit/parseUri/index.spec.js

```diff
@@ -50,4 +50,25 @@ describe('lib/parseUri', function () {
     expect(parsedUri.queryKey).to.eql({ foo: 'bar', baz: '3' });
   });
 
+  it('correctly parses a URL with the @ symbol in the query', function () {
+    var url = 'https://dev04-webstore-jackwills.demandware.net/on/demandware.store/Sites-jwuk-Site/en_GB/DebugPipelines-ShowConfirmation?cid=p@p.com&&oid=00026203';
```

Comment:
We should not promote use URLs that belong to other organizations.  Please replace with an example URL that demonstrates the problem.